src/net/http.rs

```diff
@@ -4895,9 +4895,10 @@ mod test {
             let res = HttpRequestPreamble::consensus_deserialize(&mut data.as_bytes());
             test_debug!("Expect '{}'", errstr);
             let expected_errstr = format!("{:?}", &res);
```

Comment:
could simplify to `assert!(res.is_err(), "{:?}", &res);` and delete previous line

Reply:
Addressed in 93a7c393069ca4420433e03338a6534bee19dc8f